parsl/executors/high_throughput/executor.py

```diff
@@ -323,7 +323,7 @@ def __init__(self,
             launch_cmd = DEFAULT_LAUNCH_CMD
         self.launch_cmd = launch_cmd
 
-    radio_mode = "htex"
+    radio_mode = "diaspora"
```

Comment:
I don't think you should be changing the behaviour of Parsl monitoring to use this new radio by default

Reply:
What about set this `radio_mode` in executor's `__init__` by using config file?

Reply:
yes that's probably the right way to go - change radio_mode into an object level value (self.radio_mode) instead of a class level value and set it as an `__init__` parameter